### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-08-18

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#102] [General]
This meeting will be on 8/18/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- Khronos Group presentation next Wednesday
- SIGGRAPH Update
- glTF Transform Updates
- [KHR_audio](https://github.com/KhronosGroup/glTF/pull/2137) Feedback
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
- OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- Lightmap / baking / instancing
- Show and tell
- Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
# Notes

- Presenting KHR_audio to Khronos Group next Wednesday
- SIGGRAPH
  - Possible collaboration between VRM Consortium and Khronos through MSF to do technical review of the spec
- We should push on Khronos to merge the UnityGLTF branch from Needle Tools
- Alternate path is unity asset bundle loader https://gitlab.com/lox9973/uvw.js/
- glTF Transform
  - Working on extensions in ThirdRoom https://github.com/matrix-org/thirdroom/tree/gltf-transform-instancing/src/asset-pipeline
  - We should add a function for converting nodes named `_col` or `_collider` from OnCyber / Godot to `OMI_collider`
  - https://twitter.com/donrmccurdy/status/1559860087178641408
- UnityGLTF
  - Also working on UnityGLTF extensions for lightmaps, colliders, and audio
  - Will open source the Thirdroom UnityGLTF extension code
- Lightmap Extension should not remove lights it should add tags to signify whether they should be realtime, mixed, or baked. If the runtime doesn't support lightmaps it'll fall back to realtime lighting.

**Comment by futuristudios** [comment]
video recording of call on [ipfs](https://ipfs.io/ipfs/QmUQ7isPX3EBRhSxTvJWGVkivkMxJBpUpVrKDXjwu8H7dt?filename=2022-08-18%2022-39-46.mp4).

**Comment by madjin** [comment]
While disscussing OMI_collider we started chatting about name schemes, oncyber and hyperfy are currently using such for worlds, Godot has special names like -col


## oncyber
> These components are empty objects and all the meshes in the scene need to be parented to them depending on the behavior and usage you desire for the mesh to have.

https://docs.oncyber.io/architects/getting-started
https://oncyber.io/uploader
https://gltf-viewer.donmccurdy.com/

![brave_wPHUzt8WTc](https://user-images.githubusercontent.com/32600939/185543016-b830398b-0331-41b8-9875-b5082886d097.png)

![blender_V0XoqzK4Ia](https://user-images.githubusercontent.com/32600939/185543024-34cf31d5-b316-4ff5-b71f-63dbee7fe63b.png)

![brave_OcV9bjHUSG](https://user-images.githubusercontent.com/32600939/185543041-7eb1a4d1-3b6c-451f-bdf2-06683f5c1082.png)

![brave_jSm8IuARio](https://user-images.githubusercontent.com/32600939/185543059-1844ba69-c264-442b-a23d-cb14eb40a885.png)

## hyperfy

https://docs.hyperfy.io/3d-models#how-do-i-add-colliders
![image](https://user-images.githubusercontent.com/32600939/185543453-df82c648-1437-469a-a15b-5772f2ca31c1.png)
